main.py

```
@@ -31,88 +41,17 @@ def initialize_connection() -> c.Connection:
     # Wait for client to open if it's not open already
     while True:
         try:
-            connection = c.Connection(int(SHOULD_PRINT))
+            connection = c.Connection()
             userinput.get_first_choices(connection)
             return connection
```

Ruff (0.14.0)

Consider moving this statement to an `else` block

(TRY300)